tests/run-all.go

```diff
@@ -24,7 +24,7 @@ import (
 
 var (
 	concurrentTests = 2
-	testsTimeout    = "5m"
+	testsTimeout    = "15m"
```

Comment:
If we reduce the e2e test duration with the comment above, I'd reduce this to 10m